azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/CallingFragment.kt

```diff
@@ -210,6 +215,7 @@ internal class CallingFragment :
 
     override fun onDestroy() {
         super.onDestroy()
+        backPressedCallback.remove()
```

Comment:
Why we do not call
requireActivity().onBackInvokedDispatcher.unregisterOnBackInvokedCallback(viewModel::exitComposite)
but we do call it from SetupFragment